### PR TITLE
Add writeup on issues

### PR DESCRIPTION
**Pull request by chrisullyott**
@JesseAtMonk @jstayton Would you like to check out these new guidelines for submitting Issues? I based them on Ministry Brands' priority levels and this week's discussions.

### CODE REVIEW

File: issues.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Reporting issues
```
Comment (jstayton) [inline issues.md]: This should just be "Issues".

File: issues.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+================
+
+By default, all Issues logged in Github are meant to be considered to be a _bug_, meaning, _a software problem affecting the stability of the product_. This includes functionalities that run without error, but do something unexpected which also works against the system's stability.
+
+Priority labels
```
Comment (jstayton) [inline issues.md]: All headings should be title case.

File: issues.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+---------------
```
Comment (jstayton) [inline issues.md]: Add a empty line after headings.

File: issues.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+In September 2016, Ministry Brands helped define priority levels for issues based on an overall Service Level Agreement (SLA). We use the labels P1, P2, P3 and P4 define the overall _priority_ of resolution on the issue.
+
+**P1** - Urgent. A key system component or feature is broken. These are the team's highest priority until resolved.
+
+**P2** - Important. Should be fixed in the next 2-4 weeks.
+
+**P3** - Fix eventually (within 2-3 months).
+
+**P4** - A nice-to-have; will add as time allows. Mostly includes enhancements that aren't often requested, but would drive value in the product especially if released along with other P4's.
```
Comment (jstayton) [inline issues.md]: It should just be "P4s" at the end, no apostrophe.

File: issues.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Feature requests
+----------------
+Issues should not be used as an inbox for client feature requests. Instead, each member of the team should be comfortable acting as a kind of gatekeeper for the requests that come in from the clientbase, working across the teams to determine the potential value of the feature.
+
+Requests for entirely new features should be fleshed out in a separate document, outside of Issues. The _Voice of the Customer_ meetings are the best place to flesh out your ideas with the general managers.
+
+Gray areas
+----------
+**What if something is not a _bug_, but is confusing to use?** - These are "enhancements", but can be raised in priority if necessary depending on client feedback.
```
Comment (jstayton) [inline issues.md]: I would create an "Enhancements" section that explains things, similar to how you have a "Feature requests" section.

File: issues.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+**security** - Issues that could have an effect on the privacy of client information.
+
+_Note: Adding any of these labels doesn't change the apparent priority of an issue. It is possible to have a high-priority "enhancement"._
+
+Feature requests
+----------------
+Issues should not be used as an inbox for client feature requests. Instead, each member of the team should be comfortable acting as a kind of gatekeeper for the requests that come in from the clientbase, working across the teams to determine the potential value of the feature.
+
+Requests for entirely new features should be fleshed out in a separate document, outside of Issues. The _Voice of the Customer_ meetings are the best place to flesh out your ideas with the general managers.
```
Comment (jstayton) [inline issues.md]: I think this paragraph goes beyond what needs to be known about issues. If the process were to change for feature requests, we'd likely forget to update the text here.

File: issues.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+Reporting issues
+================
+
+By default, all Issues logged in Github are meant to be considered to be a _bug_, meaning, _a software problem affecting the stability of the product_. This includes functionalities that run without error, but do something unexpected which also works against the system's stability.
+
+Priority labels
+---------------
+In September 2016, Ministry Brands helped define priority levels for issues based on an overall Service Level Agreement (SLA). We use the labels P1, P2, P3 and P4 define the overall _priority_ of resolution on the issue.
```
Comment (jstayton) [inline issues.md]: I don't think we should include the history of how we came to this. That's better left for the commit message.

File: issues.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+**support** - Issues which are tied to client feedback which would benefit from a personal response.
+
+**security** - Issues that could have an effect on the privacy of client information.
+
+_Note: Adding any of these labels doesn't change the apparent priority of an issue. It is possible to have a high-priority "enhancement"._
+
+Feature requests
+----------------
+Issues should not be used as an inbox for client feature requests. Instead, each member of the team should be comfortable acting as a kind of gatekeeper for the requests that come in from the clientbase, working across the teams to determine the potential value of the feature.
```
Comment (jstayton) [inline issues.md]: I don't think the "Instead..." sentence is necessary. Again, I think it's beyond the scope of what needs to be known about issues.

I'd instead define what a feature request is considered.

**Comment by chrisullyott** [comment]
@jstayton I made some changes and added a basic issue template. What do you think?

File: issues.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+======
+
+The MonkDev development team uses GitHub Issues for tracking bugs in products. Periodically, reports are taken from Issues to shed light on the overall performance of the platform.
+
+By default, all Issues logged in Github are meant to be considered to be a _bug_, meaning, _a software problem affecting the stability of the product_. This includes functionalities that run without error, but do something unexpected which also works against the system's stability.
+
+Priority Labels
+---------------
+
+We use the labels P1, P2, P3 and P4 define the overall _priority_ of resolution on the issue.
```
Comment (jstayton) [inline issues.md]: "to define"

File: issues.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
+Issues
+======
+
+The MonkDev development team uses GitHub Issues for tracking bugs in products. Periodically, reports are taken from Issues to shed light on the overall performance of the platform.
+
+By default, all Issues logged in Github are meant to be considered to be a _bug_, meaning, _a software problem affecting the stability of the product_. This includes functionalities that run without error, but do something unexpected which also works against the system's stability.
```
Comment (jstayton) [inline issues.md]: "meant to be considered to be"

File: issues.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
+Issues
+======
+
+The MonkDev development team uses GitHub Issues for tracking bugs in products. Periodically, reports are taken from Issues to shed light on the overall performance of the platform.
```
Comment (jstayton) [inline issues.md]: You can just say "We use...".

File: issues.md (hunk 10 above)
Comment (jstayton) [inline issues.md]: GitHub

File: issues.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+------------
+
+Though _enhancements_ are not _bugs_, they are inherently no less important; their priority is still defined with P1-P4 labels. For example, if a new feature is in working order, but part of it generates enough negative feedback, a new issue can be created as a high-priority enhancement. The issue in no way causes instability in the platform, but needs to be addressed quickly for the users.
+
+Feature Requests
+----------------
+
+Feature requests are ideas for new capabilities to be added to the product, such as:
+
+- A new module
```
Comment (jstayton) [inline issues.md]: See how we do lists in other files. It's `*` or `1.` followed by the appropriate number of spaces so the first word of the sentence starts on the fourth character. And we use a period or colon at the end of each sentence.

File: issues.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+- Functionality that one module doesn't have, where another one does
+- Integration with a third-party service
+
+Issues is not a great place for feature requests. Feature requests should be fleshed out by several team members in a separate place, like Trello or Basecamp.
```
Comment (jstayton) [inline issues.md]: I'd say it's not the place for feature requests, period.

File: issues.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+
+Issue Template
+--------------
+
+GitHub supports [Issue Templates](https://help.github.com/articles/creating-an-issue-template-for-your-repository/). Here's a template to use for the repository:
+
+```
+_Brief description_
```
Comment (jstayton) [inline issues.md]: I'd leave this out in favor of a well-constructed issue title.

File: issues.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+
+
+Current behavior
```
Comment (jstayton) [inline issues.md]: This is typically called "actual behavior", and I usually see it after "expected behavior".

File: issues.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+----------------
+
+
+Expected behavior
+-----------------
+
+
+Steps to reproduce
+------------------
+
```
Comment (jstayton) [inline issues.md]: I'd include a placeholder ordered list:

```
1.
2.
3.
4.
5.
```

**Pull request merged**